packages/expo-modules-core/ios/Swift/Methods/ConcreteMethod.swift

```diff
@@ -0,0 +1,89 @@
+
+public struct ConcreteMethod<Args, ReturnType>: AnyMethod {
+  public typealias ClosureType = (Args) -> ReturnType
+
+  public let name: String
+
+  public var takesPromise: Bool {
+    return argTypes.last?.canCast(Promise.self) ?? false
+  }
+
+  public var argumentsCount: Int {
+    return argTypes.count - (takesPromise ? 1 : 0)
+  }
+
+  public var queue: DispatchQueue?
+
+  let closure: ClosureType
+
+  let argTypes: [AnyArgumentType]
+
+  init(
+    _ name: String,
+    argTypes: [AnyArgumentType],
+    queue: DispatchQueue? = nil,
+    _ closure: @escaping ClosureType) {
```

Comment:
```suggestion
    _ closure: @escaping ClosureType
  ) {
```
I prefer this visual separation between signature and body. Without this one indents in each line are the same.